README.md

```diff
@@ -190,14 +190,14 @@ variables correctly, you should then be able to do e.g.:
 
 ```python
 >>> from changebot.github.github_api import IssueHandler
->>> issue = IssueHandler('astrofrog/test-bot', 5, installation=36238)
+>>> issue = IssueHandler('astrofrog/test-bot', 5, installation=37176)
```

Comment:
Actually the mistake is above, it should say ``36238 is the installation ID`` - can you fix and change this back?

Reply:
Ops. Done.